### PR TITLE
docs: removes context tags from bridge doc

### PR DESCRIPTION
**Pull request by PaulRMellor**
Signed-off-by: prmellor <pmellor@redhat.com>

Removes a couple of _context_ tags from the doc that aren't needed.
Also updates the API reference title and description.

### CODE REVIEW

File: src/main/resources/openapiv2.json [hunk 1]
```diff
@@ -1926,7 +1926,7 @@
     }
   ],
   "externalDocs": {
```
Comment (PaulRMellor) [inline src/main/resources/openapiv2.json:1928]: I guess we need to include _externalDocs_? Not sure if we can keep out of docs?

**Comment by PaulRMellor** [comment]
@ShubhamRwt 

Is it possible to add descriptions under the _paths_ and _definitions_ headings?
https://strimzi.io/docs/bridge/in-development/#_definitions
https://strimzi.io/docs/bridge/in-development/#_paths

**Comment by ShubhamRwt** [comment]
> @ShubhamRwt
> 
> Is it possible to add descriptions under the _paths_ and _definitions_ headings? https://strimzi.io/docs/bridge/in-development/#_definitions https://strimzi.io/docs/bridge/in-development/#_paths

Hi, I can see you are adding description under

> @ShubhamRwt
> 
> Is it possible to add descriptions under the _paths_ and _definitions_ headings? https://strimzi.io/docs/bridge/in-development/#_definitions https://strimzi.io/docs/bridge/in-development/#_paths

@PaulRMellor Hi, Do you mean right after the `PATH` header?

**Review by scholzj — COMMENTED** [review]
You have to run `make docu_api` and commit the updated API files it generates.

**Comment by PaulRMellor** [comment]
> > Is it possible to add descriptions under the _paths_ and _definitions_ headings? https://strimzi.io/docs/bridge/in-development/#_definitions https://strimzi.io/docs/bridge/in-development/#_paths
> 
> @PaulRMellor Hi, Do you mean right after the `PATH` header?

@ShubhamRwt - after the main 4.2 and .4.3 headings to introduce the sections.

**Review by scholzj — APPROVED** [review]

**Comment by ShubhamRwt** [comment]
=
> @ShubhamRwt - after the main 4.2 and .4.3 headings to introduce the sections.

I see. @PaulRMellor  I tried some ways but was unable to get the description there.

**Review by ShubhamRwt — APPROVED** [review]

**Comment by PaulRMellor** [comment]
> =
> 
> > @ShubhamRwt - after the main 4.2 and .4.3 headings to introduce the sections.
> 
> I see. @PaulRMellor I tried some ways but was unable to get the description there.

Okay. No worries.

File: src/main/resources/openapiv2.json (hunk 1 above)
Comment (PaulRMellor) [reply src/main/resources/openapiv2.json:1928]: @ShubhamRwt I think we need to retain this too, right?

**Review by ppatierno — APPROVED** [review]

**Pull request merged**